documentation/src/main/asciidoc/topics/proc_cli_adding_entries.adoc

```diff
@@ -1,35 +1,31 @@
 [id='cli_add_entries']
 = Adding Cache Entries
-Add data to caches with the {brandname} CLI.
+Create `key:value` pair entries in the data container.
 
 .Prerequisites
 
-* Create a cache named "mycache" and `cd` into it.
-+
-[source,options="nowrap",subs=attributes+]
-----
-[//containers/default]> cd caches/mycache
-----
+Create a {brandname} cache that can store your data.
 
 .Procedure
 
-. Put an entry into "mycache".
+. Create a CLI connection to {brandname}.
+. Add entries into your cache as follows:
++
+* Use the [command]`put` command from the context of a cache:
 +
 [source,options="nowrap",subs=attributes+]
 ----
 [//containers/default/caches/mycache]> put hello world
 ----
 +
-[TIP]
+* Use the [command]`--cache=` with the [command]`put` command:
 ====
```

Comment:
I think this line is breaking the HTML formating of this section.

Reply:
great catch